drivers/network/netio/CMakeLists.txt

```diff
@@ -0,0 +1,24 @@
+
+spec2def(netio.sys netio.spec ADD_IMPORTLIB)
+
+# TODO: dependency on afd.h should go away ...
```

Comment:
Is this dependency still present as of today?

Reply:
Yes unfortunately. The tdi.c and tdiconn.c helpers depend on the afd.h header (netio.c not). So I would have to fix the tdi helpers. I can do that but not today :) Please tell me what I should do about it.